lib/mixpanel-node.d.ts

```diff
@@ -68,6 +68,8 @@ declare namespace mixpanel {
     set(distinctId: string, propertyName: string, value: string | number, callback?: Callback): void;
     set(distinctId: string, propertyName: string, value: string | number, modifiers: Modifiers, callback: Callback): void;
 
+    unset(distinctId: string, propertyName: string | string[], modifiers?: Modifiers, callback?: Callback): void;
```

Comment:
Don't you also need the case where no modifiers are passed and the third argument is the callback?
https://github.com/mixpanel/mixpanel-node/blob/55fe7d241611c8f6e8860b554bf0bd1c977e7277/lib/mixpanel-node.js#L891-L893

Reply:
Yeah good catch, looks like that's also part of the API. Thanks!